shared/domains/chelonia/db.js

```diff
@@ -157,7 +168,10 @@ export default ((sbp('sbp/selectors/register', {
       await sbp('chelonia.db/set', entry.hash(), entry.serialize())
       await sbp('chelonia.db/set', getLogHead(contractID), JSON.stringify({ HEAD: entry.hash(), height: entry.height() }))
       console.debug(`[chelonia.db] HEAD for ${contractID} updated to:`, entry.hash())
-      await sbp('chelonia.db/set', `_private_hidx=${contractID}#${entryHeight}`, entry.hash())
+      await sbp('chelonia/db/setEntryMeta', contractID, entryHeight, {
+        currentHash: entry.hash(),
```

Comment:
Could we name this simply `hash`? To me that makes more sense than `currentHash`, as it's the hash for this particular entry, and it's also nicely the same length as `date` too.

Above `currentHash` would still make sense though, e.g.:

```js
let { hash: currentHash, ...serverMeta } = (
```

Reply:
Agreed that `hash` is a better name.